README.md

```diff
@@ -7,56 +7,102 @@ Vocabulary to describe networks of people and groups.
+vf:hasMember skos:prefLabel "has member"
+'''
+'''
+ex:affiliateOf rdfs:subPropertyOf vf:memberOf
+ex:hasAffiliate rdfs:subPropertyOf vf:hasMember
+ex:harvesterFor rdfs:subPropertyOf vf:memberOf
+ex:hasHarvester rdfs:subPropertyOf vf:hasMember
+'''
+
 #### Examples
 
-* [member, affiliate](http://json-ld.org/playground/#startTab=tab-expanded&json-ld=%7B%22%40context%22%3A%7B%22inverseOf%22%3A%22owl%3AinverseOf%22%2C%22label%22%3A%22skos%3AprefLabel%22%2C%22labelMap%22%3A%7B%22%40container%22%3A%22%40language%22%2C%22%40id%22%3A%22skos%3AprefLabel%22%7D%2C%22nrp%22%3A%22http%3A%2F%2Fnrp.webfactional.com%2Faccounting%2F%22%2C%22owl%22%3A%22http%3A%2F%2Fwww.w3.org%2F2002%2F07%2Fowl%23%22%2C%22rdf%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%2C%22rdfs%22%3A%22http%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23%22%2C%22skos%22%3A%22http%3A%2F%2Fwww.w3.org%2F2004%2F02%2Fskos%2Fcore%23%22%2C%22vf%22%3A%22https%3A%2F%2Fw3id.org%2Fvalueflows%2F%22%7D%2C%22%40graph%22%3A%5B%7B%22%40id%22%3A%22nrp%3Aagent-relationship-type-lod%2FisAffiliateOf%22%2C%22%40type%22%3A%22rdf%3AProperty%22%2C%22inverseOf%22%3A%7B%22%40id%22%3A%22nrp%3Aagent-relationship-type-lod%2FhasAffiliate%22%7D%2C%22labelMap%22%3A%7B%22en%22%3A%22is%20affiliate%20of%22%7D%7D%2C%7B%22%40id%22%3A%22nrp%3Aagent-relationship-type-lod%2FhasAffiliate%22%2C%22%40type%22%3A%22rdf%3AProperty%22%2C%22inverseOf%22%3A%7B%22%40id%22%3A%22nrp%3Aagent-relationship-type-lod%2FisAffiliateOf%22%7D%2C%22labelMap%22%3A%7B%22en%22%3A%22has%20affiliate%22%7D%7D%2C%7B%22%40id%22%3A%22nrp%3Aagent-relationship-type-lod%2FhasMember%22%2C%22%40type%22%3A%22rdf%3AProperty%22%2C%22inverseOf%22%3A%7B%22%40id%22%3A%22nrp%3Aagent-relationship-type-lod%2FisMemberOf%22%7D%2C%22labelMap%22%3A%7B%22en%22%3A%22has%20member%22%7D%7D%2C%7B%22%40id%22%3A%22nrp%3Aagent-relationship-type-lod%2FisMemberOf%22%2C%22%40type%22%3A%22rdf%3AProperty%22%2C%22inverseOf%22%3A%7B%22%40id%22%3A%22nrp%3Aagent-relationship-type-lod%2FhasMember%22%7D%2C%22labelMap%22%3A%7B%22en%22%3A%22is%20member%20of%22%7D%7D%5D%7D&context=%7B%7D)
+* [member, affiliate](http://json-ld.org/playground/#startTab=tab-expanded&json-ld=https%3A%2F%2Fraw.githubusercontent.com%2Fvalueflows%2Fagent%2Fmaster%2Fexamples%2Fnrp-relationshiptypes.jsonld)
 
 ### Agent
 
-#### Notes
+#### Definition and Notes
 
-An Agent is an entity that has agency in the economic realm.  This could be a person, organization, group, network, etc.  
+An Agent is an entity that has agency in the social or economic realm.  This could be a person, organization, group, network, etc.
 
 #### Properties
 
 * @id ([JSON-LD](http://www.w3.org/TR/json-ld/) specific way of setting [rdf:subject](http://www.w3.org/TR/rdf-schema/#ch_subject))
 * @type ([JSON-LD](http://www.w3.org/TR/json-ld/) specific alias of [rdf:type](http://www.w3.org/TR/rdf-schema/#ch_type))
-* url 
-* displayName / displayNameMap
-* image
-* (will be others)
+* url (a URL)
+* displayName (skos:prefLabel)
```

Comment:
@ahdinosaur thanks, was having a hard time finding final decisions on some of these, makes sense it is in the context.  Get it on note.  

On label, did we decide to use label for people's names as well as group names?  Doesn't seem right to me now.  I get it if everything is just a graph on a page, but seems not so good for a domain vocab.  (Sorry probably am bike shedding again.... or maybe all vocab work is that way? :)  )